A2rchi/interfaces/chat_app/static/script.js-template

```diff
@@ -4,10 +4,18 @@ const chatContainer = document.querySelector(".chat-container");
 const themeButton = document.querySelector("#theme-btn");
```

Comment:
See comments I left on `script.js` re: adding `message_id` and submitting dislike feedback on closing of popup window.